pkgs/servers/livekit/cli.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "livekit-cli";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "livekit";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-euA+MDCqUTAJ4X9D+XQWAs8HKnWOh+KclTdpoQHGADk=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.